src/config/ConfigService.ts

```diff
@@ -14,24 +14,20 @@ import {
   getLogControl,
   LogGroupControlSettings,
 } from 'typescript-logging'
+import { ERROR_WS_ADDRESS_NOT_SET } from '../errorhandling/SDKErrors'
 
-// Create options instance and specify 1 LogGroupRule:
-// * LogLevel Error on default, env DEBUG = 'true' changes Level to Debug.
-const options = new LoggerFactoryOptions().addLogGroupRule(
-  new LogGroupRule(
-    new RegExp('.+'),
-    process.env.DEBUG && process.env.DEBUG === 'true'
-      ? LogLevel.Debug
-      : LogLevel.Error
-  )
-)
-// Create a named loggerfactory and pass in the options and export the factory.
-// Named is since version 0.2.+ (it's recommended for future usage)
-// eslint-disable-next-line import/prefer-default-export
-export const factory = LFService.createNamedLoggerFactory(
-  'LoggerFactory',
-  options
-)
+const { DEFAULT_WS_ADDRESS } = process.env
```

Comment:
I remember Timo saying that we want to remove endpoint configuration by environment variables. I'd keep this option in integration tests, but this could be solved by calling `Kilt.config()` in each integration test file (or in a file they all import, like `utils`